### ISSUE TITLE
Manage Email Notifications with Queues

### ISSUE THREAD

**Issue by poppastring** [poppastring/dasblog-core#636]
When adding comments, we have the options to send emails. These emails should be managed via queues not synchronously in the web request. :(

**Comment by clemensv** [comment]
I agree. The notification handlers for entries use the thread pool queue for that. 

Also see #671. We should also raise events for comments and allow for the email notifications (etc.) to be handled externally, e.g. in Logic Apps/Power Automate if someone would like to use that.
